Vagrantfile

```diff
@@ -41,6 +41,13 @@ Vagrant.configure("2") do |config|
     end
   end
 
+  config.vm.provider :libvirt do |l,override|
+    override.vm.box = "generic/ubuntu2204" # un
+    l.memory = 2048
+    override.vm.provision "shell", path: "deployment/provision_vagrant_vm.sh"
+    override.vm.allowed_synced_folder_types = ["sshfs"]
+  end
+
```

Comment:
I think we wouldn't want to merge libvirt support (for now) -- can we exclude this part from the PR?

(You can keep using it locally, but I wouldn't want unsupported code in the main repo)